### ISSUE TITLE
Tunnelblick freezes when showing VPN details for the first time after launch.

### ISSUE THREAD

**Issue by JPaulMora** [Tunnelblick/Tunnelblick#762]
**Describe the bug**
Whenever I want to show the config screen, Tunnelblick loads for a few minutes (spinning wheel over tray icon) and eventually shows. I always wait about 5 minutes.

**To Reproduce**
1. Go to header tray icon
2. Click on VPN Details..
4. See error (just wait till it unfreezes)

**Expected behavior**
Should open quickly

**Screenshots**
<img width="1088" alt="image" src="https://github.com/Tunnelblick/Tunnelblick/assets/8335656/fe6a4a71-d8d7-4a14-8688-ef3bf62e4294">

**Desktop (please complete the following information):**
 - OS: macOS Ventura 13.4.1 

**Additional context**

**Comment by jkbullard** [comment]
Are you running opencore legacy patcher? See the (long) discussion at

     https://groups.google.com/g/tunnelblick-discuss/c/zJ81-igy1rA

In particular, my comment at

     https://groups.google.com/g/tunnelblick-discuss/c/zJ81-igy1rA/m/e_DqOCSGAAAJ

(There does not appear to be a way for Tunnelblick to fix this; it seems to be a crash in or caused by OCLP.)

**Comment by JPaulMora** [comment]
Yes! I'm on Ventura MBP from 2014. I can confirm that's the issue, on the crash logs there's one crash per Tunnelblick setting.

Hope OCLP people can help out eventually.

**Comment by Danrancan** [comment]
> Yes! I'm on Ventura MBP from 2014. I can confirm that's the issue, on the crash logs there's one crash per Tunnelblick setting.
> 
> Hope OCLP people can help out eventually.

I am experiencing the exact same proble, and I am also using OCLP on a Macbook Pro 11,3 (2014), which I believe is the same model as yours. However, I am only using MacOS Monterey 12.7.1.  Did you find a workaround or fix for this at all? Have you made a post in the OCLP issues yet?

**Comment by pradorocchi** [comment]
The cause is this error:  "attributedStringFromHTML: initWithHTML failed"

line 504 of /tunnelblick/helper.m

and tunnelblick freezes after it.. 


I had extensivelly tested it.. installed and uninstalled opencore patcher, multiple MacOs Versions. 
the problem relies on the video driver installed by OpenCore. 

If I remove the (patched) video driver, tunnelblick works fine again. 
Reinstall patched video driver, and this error occurs.

the erros just affect tunellblicks "VPN Details" menu.. 


I will try to fix tunnelblick on the next days, and I will provide feeback here. 




![Image](https://github.com/user-attachments/assets/cc0f3b0e-b568-4fcb-a8d3-f1e95b09bd3b)

**Comment by jkbullard** [comment]
@pradorocchi - Thanks for your work on this!

Although it appears this failure only occurs when Tunnelblick's "VPN Details" is clicked, I believe that **any** use of NSAttributedString's instance method `initWithHTML:options:documentAttributes:` fails, and Tunnelblick uses it in many other places, primarily in error messages, so the problem could show up unexpectedly.

The problem that that particular instance method, which normally returns an NSAttributedString, returns nil when it is invoked under OCLP.

I don't see how that can be a problem in Tunnelblick, and I'm not aware of an alternative instance method that we could use. (And it would be a lot of work because the method is used in scores, if not hundreds, of places.)

**Comment by derelvis** [comment]
> the error just affect tunellblicks "VPN Details" menu.. (beside this menu, tunnelblick works fine and connect to VPN normally) it is just the "VPN Details" menu that fails.. and the error is line 504 of helper.m
> 
> I will try to fix tunnelblick on the next days, and I will provide feeback here.
> 
> ![Image](https://github.com/user-attachments/assets/cc0f3b0e-b568-4fcb-a8d3-f1e95b09bd3b)

Experiencing the same with OCLP and wondering if you ever found a solution? Thanks in advance!

**Comment by tomelgato** [comment]
Same problem here with OCLP since ever. I agree that it may be a strange problem with "initWithHTML:options:documentAttributes:" but what makes me wonder is that Tunnelblick is the only program im aware of having these problems with OCLP.
